docs/en/reference/advanced-configuration.rst

```diff
@@ -184,6 +184,24 @@ For development you should use an array cache like
 ``Symfony\Component\Cache\Adapter\ArrayAdapter``
 which only caches data on a per-request basis.
 
+Nullability detection (***RECOMMENDED***)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+    Since ORM 3.4.0
+
+.. code-block:: php
+
+    <?php
+    $config->setInferPhpNullability(true);
+
+Sets whether Doctrine should infer the nullability of PHP types to the
+database schema. This is useful when using PHP 7.4+ typed properties
```

Comment:
```suggestion
Sets whether Doctrine should infer columns nullability from PHP types declarations.
```

I don't think it worth mentioning 7.4+, this branch requires PHP 8.1, so I doubt anyone is restraining themselves to untyped properties.